### DISCUSSION TITLE
Security: Validating a job posting

### DISCUSSION THREAD

**Discussion by carlomigueldy** [Developer-DAO/DAO-job-board#108] [Ideas]
### Overview

While having a random chat with @PBillingsby we were talking about how people could potentially flood the job board using bots, they can create a new wallet and flood more with not useful information thus that might not give us a good reputation or perspective for our users that our job board is spammy with scammy postings. Tackling this early on or finding a solution would be great.

### Solution

@PBillingsby suggested to have an admin dashboard where we can manage job postings, that we can maybe ban or suspend users however this approach can make the job board a bit more centralized since we're controlling what users would see. But it's for creating a good user experience for our users that we would prevent scammy-spammy postings. 

It would be cool that if we can come up with an algorithm that filters or prevents this kind of problems but it might be difficult. 

However if you might have a better suggestion on how we go about it please comment down below, we'd love to discuss it with you.

**Comment by carlomigueldy** [comment]
Issue of origin https://github.com/Developer-DAO/DAO-job-board/issues/84

**Comment by PBillingsby** [comment]
Another suggestion is have the poster paste a randomly generated word to post and match them before allowing the POST call?

**Comment by carlomigueldy** [comment]
> @angeljgomezc @carlomigueldy A thought I had was wouldn't a job poster have to pay gas for a post? This might deter a lot of spam bots based on this alone.

Yeah I think paying gas would work, since job board sites charge $249 to create a job post.

cc @angeljgomezc @PBillingsby
